docs/src/guide/query-methods.md

```diff
@@ -172,16 +172,24 @@ const records = db.table
 
 When there is a need to use a piece of raw SQL, use the `sql` method.
 
-To select with a raw SQL, need to specify a column type as a first argument, so the TS could use it to guess the result type of the query:
+To select with a raw SQL, need to specify a column type as a first argument, so the TS could use it to infer the result type of the query:
 
 ```ts
-const result: { num: number }[] = await db.table.select({
+const result = await db.table.select({
   num: db.table.sql((t) => t.integer())`
     random() * 100
   `,
 });
 ```
 
+Alternatively, static-cast the raw expression with:
```

Comment:
Here it's important to highlight the difference. `castTo` is only specifying the type, `(t) => t.integer()` has some additional bonuses as I mentioned in this PRs thread.

Reply:
`static-cast` - google leads to  C++ static_cast, avoid this term pls. I'd say "Alternatively, specify the type for raw express with:".